README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Introduction
+
+My name is Aidan Coffey and I am from Long Island, NY. I am a 4th in the College majoring in Political Science and Public Policy. (*BA Spring 2020*). 
+
+## Politcal Science Background
+
+At UChicago, I have focused my study on American politics. I have supplanted that academic focus while organizing for the Missouri Democratic Party. I recommend you check out their website here! <https://missouridemocrats.org/>
+
+At the MO-DP, I..
+* knocked at least 100 doors a day
```

Comment:
Use of list emphasizes the fact that you had multiple responsibilities! Love it.